emission/core/get_database.py

```diff
@@ -15,7 +15,13 @@
 config = ecbc.get_config('conf/storage/db.conf',
     {"DB_HOST": "timeseries.url", "DB_RESULT_LIMIT": "timeseries.result_limit"})
 
-print("Retrieved config %s" % config)
+db_config = {}
+for key in ["DB_HOST", "DB_RESULT_LIMIT"]:
+    if key in config:
+        db_config[key] = config.get(key)
+    else:
+        db_config[key] = None
+print("Retrieved config: %s" % db_config)
```

Comment:
I really don't like the approach of changing code to make the tests pass, particularly when this is not even critical to the functionality exercised by the failing test.

The test that is failing really needs to check whether the script is working properly or not.

To rephrase, the previous logs actually have other benefits in terms of logging - for example, they can tell us whether the variables came from the environment (there are other variables) or from the config file (there are not).

This may even be meaningful if this was affecting multiple tests.

But this is really only affecting one test, which doesn't even care about the values of the variables, and only cares about the operation of the script with various input options. So changing the code doesn't appear to be the correct approach to fix this.